docs/detections/alerts-view-details.asciidoc

```diff
@@ -22,29 +22,54 @@ image::images/alert-details-flyout.png[Alert details flyout]
-
-* *Alert prevalence*: The total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for `host.name` means three alerts with the same `host.name` value exist within the timeframe.
+* *Alert prevalence*: Shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for `host.name` means three alerts with the same `host.name` value exist within the timeframe.
 +
 Alert prevalence data can help you investigate relationships with other alerts and gain context about the events producing alerts. You can also click the alert prevalence count to explore the alerts in Timeline.
 +
 IMPORTANT: Before investigating alert prevalence data in Timeline, save any Timelines you're working on to ensure you can access them later.
 
-* *Enriched data*: Available threat indicator matches and threat intelligence data. This section only displays for alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
+* *Insights*: Shows relationships with related alerts to help you quickly identify patterns. Refer to <<alert-details-insights, Insights on alerts>> for more information.
+
+* *Enriched data*: Displays available threat indicator matches and threat intelligence data. This section only displays for alerts with intelligence data. Click the info icon to learn more about what data is collected. Refer to <<enriched-data-overview>> for more information.
 +
 [role="screenshot"]
 image::images/enriched-data-info-icon.png[Informational message on enriched data, 600]
 
+[discrete]
+[[alert-details-insights]]
+==== Insights on alerts
+
+The Insights section shows you how an alert is related to other alerts and offers ways to investigate related alerts. You can use this information to quickly find patterns between alerts and then take action.
+
+You can click on the section title for each insight to expand or collapse it. Some sections are only offered with a certain license and might require you to also enable a feature flag.
+
+[role="screenshot"]
+image::images/insights-section.png[Informational message on enriched data, 600]
+
+The Insights section provides the following details:
+
+* *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
+* *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find other alerts that were generated by the same or different rules. You can use this information to gain more context about the source event creating the alerts. Click the *Investigate in timeline* button to examine related alerts in Timeline.
```

Comment:
```suggestion
* *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
```